modules/tinymce/changelog.txt

```diff
@@ -2,6 +2,7 @@ Version 5.4.0 (TBD)
     Added all table menu items to the UI registry, so they can be used by name in other menus #TINY-4866
     Changed `mceInsertTable` command and `insertTable` API method to take optional header rows and columns arguments #TINY-6012
     Fixed filters for screening commands from the undo stack to be properly case-insensitive #TINY-5946
+    Fixed fullscreen plugin now removes all classes when editor is closed #TINY-4048
```

Comment:
We missed this in your previous PR sorry, but "fullscreen" is the name of a plugin here, so it should be in backticks. Also there's a `the` missing before `editor`:
```
Fixed `fullscreen` plugin now removes all classes when the editor is closed #TINY-4048
```